src/components/App/index.tsx

```diff
@@ -66,14 +66,14 @@ export const App: FC = () => {
           path="/"
           exact
           isLoggedIn={!!loginToken}
-          newUserCheck={newUserCheck}
+          newUserCheck={newUserCheck && !!whoAmI?.courses.length}
```

Comment:
in 31 line - `const newUserCheck = !!whoAmI?.courses.length;`
in 69 line newUserCheck={newUserCheck)